src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -2,17 +2,15 @@
 
 public class BooleanExpression {
 
-    /**
+    /*
      * Реализуйте метод booleanExpression(boolean a, boolean b, boolean c, boolean d),
      * возвращающий true, если среди четырех его аргументов ровно два истинны (любые)
      * во всех остальных случаях метод должен возвращать false.
-     *
      * Вам надо только проанализировать переданные в метод booleanExpression
-     * значения (a, b, c, d) и вернуть результат.
-     *
+     * значения (a, b, c, d) и веmрнуть результат.
      * Это задание необходимо решать с использованием логических операторов.
      */
-    public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+    protected boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
+        return !(a & b & c & d) && !(a ^ b ^ c ^ d) && !(a || b || c || d);
```

Comment:
does this code work correctly with value a = true b = true c = false d = false ?